xbmc/video/VideoInfoScanner.cpp

```diff
@@ -1161,6 +1162,14 @@ namespace VIDEO
                   CURL::GetRedacted(episode.strPath), episode.cDate.GetAsLocalizedDate(),
                   expression[i].regexp);
       }
+      else if (byTitle)
+      {
+        if (!GetEpisodeTitleFromRegExp(reg, episode))
+          continue;
+
+        CLog::Log(LOGDEBUG, "VideoInfoScanner: Found title based match %s (%s) [%s]", CURL::GetRedacted(episode.strPath).c_str(),
```

Comment:
Please update to the current log standards using `{}`.

For more information please see the [libfmt documentation](https://fmt.dev/latest/syntax.html)